src/src/Tunnel/TunnelRunner.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace QIT_CLI\Tunnel;
+
+use QIT_CLI\Environment\Docker;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class TunnelRunner {
+	/** @var Docker */
+	protected $docker;
+
+	/** @var OutputInterface */
+	protected $output;
+
+	public function __construct( Docker $docker, OutputInterface $output ) {
+		$this->docker = $docker;
+		$this->output = $output;
+	}
+
+	public function start_tunnel( string $local_url, string $env_id ): string {
+		if ( ! empty( getenv( 'QIT_TUNNEL_REGION' ) ) ) {
```

Comment:
> Wonder how can I define and test the region?

This is kind of a hidden setting that I'm still wondering how to approach, it's currently an env var:

`QIT_TUNNEL_REGION=us qit env:up --tunnel`

As for the message above, in the screenshot, you actually shouldn't add that line to your hosts file for tunnelled connections, I've just pushed a commit to not show the "add to hosts file" message for tunnelled connections.

As to why it's not working for you I'd have to see. Do you have a few mins to spare for a screenshare session?

Reply:
@rcstr I realized that tunnelling through Docker won't work on Mac because it requires network mode set to `host`, which is only available in Linux. I've added a workaround, which is using a local tunnel that can be easily installed with one command. Can you please try it out again and see how it goes? I won't give too much detailed instructions as the CLI instructions should be enough - I think it's part of the review that the info in there is enough.

Reply:
PS: Just remember the checkout the Manager branch and point your CLI to local